client/src/components/AboutBanner/AboutBanner.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+import "./AboutBanner.css"
+import { Jumbotron, Button, Image } from 'react-bootstrap/lib';
+import { Container, Row } from "../Grid"
+
+
+class AboutBanner extends React.Component {
+
+    render() {
+        return (
+            <Jumbotron xs={12} md={8}>
+                <Container>
+                    <Row>
+                        <h1>Connect with other brewers.</h1>
+                    </Row>
+                    <Row>
+                        <Image xs={6} md={6} lg={6} src='https://images.unsplash.com/photo-1515303900226-036e8b860207?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=83b7b7ecf6d99a4b2742f825fe0d2cf4&auto=format&fit=crop&w=600&q=60' alt='glass of beer' rounded responsive />
```

Comment:
You are defining all screens to display the same number of columns, this is redundant. All you have to do is define:
`xs={6}`
This will apply to xs screens and above.